lms/views/api/blackboard/authorize.py

```diff
@@ -0,0 +1,34 @@
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import view_config
+
+from lms.services import NoOAuth2Token
+from lms.validation.authentication import CanvasOAuthCallbackSchema
+
+
+@view_config(
+    request_method="GET", route_name="blackboard_api.oauth.authorize", permission="api"
+)
+def authorize(request):
+    return HTTPFound(
+        location=request.route_url(
+            "blackboard_api.oauth.callback",
+            _query={"state": CanvasOAuthCallbackSchema(request).state_param()},
```

Comment:
Calling `CanvasOAuthCallbackSchema` from this Blackboard view is obviously weird. We need to either add a `BlackboardOAuthCallbackSchema` or rename `CanvasOAuthCallbackSchema` to just `OAuthCallbackSchema` but until we implement the proper Blackboard API authorization I don't know which yet: I don't know how similar Blackboard's OAuth callbacks are to Canvas's (but I'm guessing pretty similar)